python/orca/test/bigdl/orca/learn/ray/pytorch/test_estimator_pyspark_backend.py

```diff
@@ -709,5 +709,34 @@ def test_optional_model_creator(self):
 
         assert np.array_equal(expected_result, result_after)
 
+    def test_jit_model(self):
+        sc = init_nncontext()
+        spark = SparkSession.builder.getOrCreate()
+        rdd = sc.range(0, 100)
+        data = rdd.map(lambda x: (np.random.randn(50).astype(np.float).tolist(),
+                                  [float(np.random.randint(0, 2, size=()))])
+                       )
+        schema = StructType([
+            StructField("feature", ArrayType(FloatType()), True),
+            StructField("label", ArrayType(FloatType()), True)
+        ])
+
+        df = spark.createDataFrame(data=data, schema=schema)
+
+        def get_jit_model(config):
+            torch.manual_seed(0)
+            return torch.jit.script(Net())
```

Comment:
https://github.com/intel-analytics/BigDL/blob/main/python/orca/example/learn/pytorch/resnet50/inference.py#L285
Can we use something similar like here to simulate the case when this issue occurs?